src/curl.c

```diff
@@ -281,7 +282,7 @@ double swupd_curl_query_content_size(char *url)
 		return -1;
 	}
 
-	curl_ret = curl_easy_setopt(curl, CURLOPT_HEADER, 0L);
+	curl_ret = curl_easy_setopt(curl, CURLOPT_HEADER, 1L);
```

Comment:
Wont this include the size of the header in the calculation of the total size? is this expected?

Reply:
yeah it looks like this is not necessary, I was under the impression that we needed to include the headers because the file size was actually retrieved from the headers, but I see now that even when changing this to 0L the file size is still retrieved correctly. I will leave the value as 0L (which is the default).